app/src/main/java/org/mozilla/focus/session/ui/SessionsSheetFragment.java

```diff
@@ -102,10 +102,12 @@ public void onAnimationEnd(Animator animation) {
         animator.addListener(new AnimatorListenerAdapter() {
             @Override
             public void onAnimationEnd(Animator animation) {
-                getActivity().getSupportFragmentManager()
-                        .beginTransaction()
-                        .remove(SessionsSheetFragment.this)
-                        .commit();
+                if (getActivity() != null) {
+                    getActivity().getSupportFragmentManager()
```

Comment:
Even if the first call returns a non null activity, the second call can still return null.

Reply:
@pocmo The first call returns a non null activity, and hence the code under the 'if' statement is executed, but the second call returns a null, and hence nothing is done, preventing the crash.
This seems to be working fine as per my testing.

Reply:
If the second call returns null then this would be a NullPointerException again.

Don't you want to do something like this:

```java
final Activity activity = getActivity();
if (activity != null) {
    activity.getSupportFragmentManager() ....
    ...
}
```